zephyr/CMakeLists.txt

```diff
@@ -704,6 +704,10 @@ zephyr_library_sources_ifdef(CONFIG_COMP_COPIER
 	${SOF_AUDIO_PATH}/copier/copier_dai.c
 )
 
+zephyr_library_sources_ifdef(CONFIG_PCM_REMAPPING_CONVERTERS
+	${SOF_AUDIO_PATH}/pcm_converter/pcm_remap.c
+)
```

Comment:
this is not required anymore, `src/audio/pcm_converter/CMakeLists.txt` should be enough.

Reply:
Nope, I receive "undefined reference" if trying to build without these lines in zephyr/CMakeLists.txt. Seems src/audio/pcm_converter/CMakeLists.txt is ignored for builds with Zephyr. Or maybe some other target should be used in that file, not "sof" as it is used now for other sources there?

Reply:
Sorry, the `audio/` subdirectory has not been added yet.

But keep at least all `pcm_converter/` files together.

Reply:
> But keep at least all pcm_converter/ files together.

Other pcm_converter sources are added in big unconditional zephyr_library_sources where all common sources are added. pcm_remap.c is added under the flag CONFIG_PCM_REMAPPING_CONVERTERS so I put it among other sources added under its flags.

Reply:
Good point, thanks.

The "big unconditional `zephyr_library_sources`" was a quick hack to get started and never a good idea. Can you please extract the other pcm_converter/ files and move them here, close to this new one? Still unconditionally.

1. The long term goal is to migrate ALL these files down to subdirectories. Keeping close files together helps.

2. Even before any migration, it's very confusing to have `pcm_converter/` (or any other subdirectory) scattered across this giant file.

Thanks for your patience!